CMake/External_OpenCV.cmake

```diff
@@ -82,6 +82,24 @@ else()
 endif()
 
 
+if(fletch_ENABLE_OpenBLAS)
+  if (NOT OpenCV_version VERSION_LESS 3.2.0)
+    message(STATUS "OpenCV depending on fletch OpenBLAS")
+    set(_OpenCV_ENABLE_OPENBLAS_DEFAULT TRUE)
+    list(APPEND OpenCV_DEPENDS OpenBLAS)
+    list(APPEND OpenCV_EXTRA_BUILD_FLAGS
+      # FIXME: If LAPACK is on, opencv_traincascade fails to link.
+      # The error is: liblapack.so - undefined reference to `gotoblas`.
+      # note: the problem may be multiple cblas.h
+      # A temporary workaround is to turn LAPACK off.
+      -DWITH_LAPACK:BOOL=FALSE
```

Comment:
If anyone has a guess as to what is causing this issue I'd love to know. Otherwise this workaround is probably fine for the time-being.

Reply:
What is lapack providing to OpenCV? Just wondering what we're losing by disabling. One option we'd considered was having fletch provide lapack, which I think is more a larger suite than a single tool, but I can't recall at the moment.

Reply:
I don't know, but I'd like to know the answer as well. I think OpenBLAS contains the LAPACK (Linear Algebra PACKage) functions. I'm not sure if this completely disables BLAS or just the LAPACK functions.  I'm really not sure what the exact structure of the BLAS+LAPACK libraries are. Does LAPACK exist as a standalone library or does it always exist as a part of a BLAS (e.g. OpenBLAS, ATLAS, MKL) package?

Reply:
I am not able to reproduce the issue with gotoblas. Is there another PR that deals with the OpenBLAS addition? I would like to work on that part first and see if we can figure out what's going on with the error.